packages/p2p/src/components/my-profile/my-profile.scss

```diff
@@ -104,10 +135,41 @@
             width: 564px;
         }
     }
+    &__privacy-setting {
+        & > .dc-toggle-switch:checked + .dc-toggle-switch__label .dc-toggle-switch__button {
+            left: calc(100% - 0.2rem);
```

Comment:
Is this a specific fix for our usage or should this be applied to `toggle-switch.jsx`?

Reply:
It's a specific applied for changing toggle switch but as  Carol left the comment we should not apply toggleswitch for now and I remove all the things related to toggleswitch